src/app/components/Header/Header.tsx

```diff
@@ -7,15 +7,34 @@ import HeaderCarousel from "./HeaderCarousel/HeaderCarousel";
 import { asText } from "@prismicio/helpers";
 import { client } from "../../../../prismicio";
 
-interface IntroPicItems {
+interface IntroPicItem {
   id: string;
   image: { url: string; alt?: string };
   heading: string;
 }
+const bioText = `Over a cup of steaming chai, some of the best stories come to life —
```

Comment:
This bio-text should be fetched from Prismic as well - so that client can change it anytime she wants.